gdnative-core/src/init.rs

```diff
@@ -0,0 +1,46 @@
+use crate::core_types::GodotString;
+
+pub struct InitCallbackInfo {
+    pub in_editor: bool,
+    pub active_library_path: GodotString,
```

Comment:
I guess it's better to make these fields private and add accessors for them?

Reply:
Would there be a benefit? If the consumer modifies them, there is no harm.

Reply:
There is harm, because there's no point in modifying them, and being able to do so is confusing.

Also we might not want to make a strong promise about their types/existence, etc.

Reply:
It's always public fields that need justification. Private fields are the default.